expat/lib/xmltok.c

```diff
@@ -42,15 +42,13 @@
 #  include <stdbool.h>
 #endif
 
-#ifdef _WIN32
+#if defined(_WIN32)
```

Comment:
Does this change come from the upstream sources?

Reply:
No, I just replaced it so it is consistent with `#elif defined()`. Should I revert it?

Reply:
I can change it to the following. Then there are only two extra lines compared to upstream.
```c++
#ifdef _WIN32
#  include "winconfig.h"
#else
#  ifdef HAVE_EXPAT_CONFIG_H
#    include <expat_config.h>
#  else
#    include "macconfig.h"
#  endif
#endif /* ndef _WIN32 */
```

Reply:
Let's do it like this, thanks!